README.md

```diff
@@ -1,13 +1,27 @@
-# Grafana LLM App (Experimental)
+# Grafana LLM App (Public Preview)
 
-A Grafana plugin designed to centralize access to LLMs, providing authentication, rate limiting, and more.
+A Grafana plugin designed to centralize access to LLMs, providing authentication, proxying, streaming, and custom extensions.
 Installing this plugin will enable various pieces of LLM-based functionality throughout Grafana.
 
-Note: This plugin is **experimental**, and may change significantly between
-versions, or be deprecated completely in favor of a different approach based on
-user feedback.
+Note: The Grafana LLM App plugin is currently in [Public preview](https://grafana.com/docs/release-life-cycle/). Grafana Labs offers support on a best-effort basis, and there might be breaking changes before the feature is generally available.
 
-## Installing this plugin
+## Install the plugin on Grafana Cloud
+
+Prerequisites:
+- Any Grafana Cloud environment (including Free)
+- API connection details from an account with [OpenAI](https://platform.openai.com) or [Azure OpenAI](https://oai.azure.com/)
+
+Steps:
+1. In your Grafana instance, open Administration → Plugins
+1. Select "All" instead of "Installed" and search for "LLM"
```

Comment:
All steps have number 1 here, should we fix this numbered list?

Reply:
This is a common pattern - Markdown renders them all correctly either way, and it's easier to just maintain a list of `1.`s than constantly changing them as you add/remove things from the list 😂